src/main/java/com/dabsquared/gitlabjenkins/trigger/handler/pipeline/PipelineHookTriggerHandlerFactory.java

```diff
@@ -14,7 +14,7 @@ private PipelineHookTriggerHandlerFactory() {}
 
     public static PipelineHookTriggerHandler newPipelineHookTriggerHandler(boolean triggerOnPipelineEvent) {
         if (triggerOnPipelineEvent) {
-            return new PipelineHookTriggerHandlerImpl(retrieve(triggerOnPipelineEvent));
```

Comment:
I don't see much of an improvement in this code change suggestion. In fact, if we would like to refactor in the future and not sure what this new `true` is referring to it is dangerous.